### DISCUSSION TITLE
Aspire test app creating temp files

### DISCUSSION THREAD

**Discussion by haldiggs** [microsoft/aspire#5681] [Q&A]
I hope this is the right place to ask this question. Is anyone else seeing randomly named files being created while developing/working in Aspire projects? Image attached. I can delete them, but they just keep coming back. Could there be some setting I have enabled that I was not aware of?

Right now I am just working through some Aspire lessons, but I wonder how this is going to be absorbed when the projects are attached to a GitHub project. I did not want to report any issues at this point.

![image](https://github.com/user-attachments/assets/3ddc1f7e-e732-47c8-9f92-09add294d61c)

**Comment by davidfowl** [comment]
There shouldn’t be any temp files under the project folder. What are in these temps files?

**Comment by haldiggs** [comment]
Agreed. Very strange. But each one of the test projects is generating them
as I code and compile. They don't seem to have anything in them. None of
them have code in them. It just continues to generate them. And it only
seems to be the aspire project. Maybe it's only the aspire test template.
For all I know.
